### PR TITLE
✨ Vendor Registration: Integrating BidGear Ad Networks into amp-ad

### PR DESCRIPTION
**Pull request by tindvubg**
✨ Vendor Registration: Integrating BidGear ad networks into AMP
For more than 8 years, BidGear has been transforming the way marketing is conducted for Brands and Consumers. Through real-time bidding, yield management, and workflow automation, BidGear enables publishers and media buyers to evaluate the attributes of each impression to determine its precise value, optimizing yield and maximizing revenue. We have seen strong growth of the AMP web platform, so we want to expand the AMP advertising platform to provide monetization opportunities for advertising partners, web developers, and users. Therefore, we would like to ask you to please approve us as soon as possible so that we can quickly participate in the advertising market on AMP. 
Thank you!

### CODE REVIEW

**Comment by tindvubg** [comment]
@calebcordry Please support me!

File: ads/_config.js [hunk 1]
```diff
@@ -1573,6 +1573,11 @@ const adConfig = jsonConfiguration({
   },
 
   'sulvo': {},
+
```
Comment (powerivq) [inline ads/_config.js:1576]: Nit: please order it alphabetically.

File: examples/ads.amp.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<!doctype html>
```
Comment (powerivq) [inline examples/ads.amp.html]: What is this file used for? I think you might want to remove this file.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: ads/_config.js (hunk 1 above)
Comment (powerivq) [reply ads/_config.js:1581]: Nit: remove this newline (line 1581)

**Review by powerivq — CHANGES_REQUESTED** [review]

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
Need to fix the lint error: https://app.circleci.com/pipelines/github/ampproject/amphtml/30644/workflows/1ead9dfc-009f-4b9e-bee6-40527f919b47/jobs/661308?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

**Comment by tindvubg** [comment]
> Need to fix the lint error: https://app.circleci.com/pipelines/github/ampproject/amphtml/30644/workflows/1ead9dfc-009f-4b9e-bee6-40527f919b47/jobs/661308?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

I have fixed it. Please let me know what I should do next. Thank you! @powerivq

**Pull request merged**